data/korath/korath.txt

```diff
@@ -223,15 +251,86 @@ fleet "Korath Home"
 	cargo 3
 	personality
 		timid opportunistic
-	variant 12
+	variant 7
 		"Korath World-Ship"
-	variant 12
+	variant 22
 		"Korath World-Ship B"
-	variant 12
+	variant 7
 		"Korath World-Ship C"
-	variant 12
+	variant 3
+		"Korath Dredger"
+		"Korath Chaser" 2
+		"Korath Chaser (Expeller)" 2
+	variant 1
+		"Korath Dredger (Shunt)"
+		"Korath Chaser" 2
+		"Korath Chaser (Expeller)" 2
+	variant 1
+		"Korath Dredger (Heaver)"
+		"Korath Chaser" 4
+	variant 1
 		"Korath Dredger"
+		"Korath Raider (Blaze)"
 		"Korath Chaser" 4
+		"Korath Chaser (Expeller)" 2
+
+fleet "Korath Large Raid"
+	government "Korath"
+	names "korath"
+	fighters "korath"
+	cargo 1
+	personality
+		disables plunders opportunistic harvests
+	variant 5
+		"Korath Raider" 2
+		"Korath Chaser"  4
+	variant 5
+		"Korath Raider (Blaze)" 2
+		"Korath Chaser" 4
+	variant 2
+		"Korath Dredger"
+		"Korath Raider (Blaze)"
+		"Korath Chaser" 6
+
+fleet "Korath Ember Waste Raid"
+	government "Korath"
+	names "korath"
+	fighters "korath"
+	cargo 1
+	personality
+		disables plunders opportunistic harvests
+	variant 30
+		"Korath Raider (Ember)"
+		"Korath Chaser" 2
+	variant 6
+		"Korath Raider (Ember Shunt)"
+		"Korath Chaser" 2
+	variant 4
+		"Korath Dredger"
+		"Korath Chaser" 4
+	variant 4
+		"Korath Dredger (Shunt)"
+		"Korath Chaser" 4
```

Comment:
Same as the normal EW raid, perhaps break out the blaze pike here as well?

Reply:
Same as above/